lib/cc/analyzer/engine.rb

```diff
@@ -60,7 +60,7 @@ def container_options
           "--net", "none",
           "--rm",
           "--volume", "#{@code_path}:/code:ro",
-          "--volume", "#{config_file}:/config.json:ro",
+          "--volume", "#{config_file.local_path}:/config.json:ro",
```

Comment:
What do you think of renaming `local_path` to `host_path`? When scanning other lines, reading "local" prompted the question "local to what?".